Examples/ByFeature/FunctionalInjection/ShoppingSteps.fs

```diff
@@ -3,15 +3,32 @@
 open NUnit.Framework
 open TickSpec
 
-type Wallet = WalletDollars of int
+type CatalogItem = {Id: int; Description: string; Price: decimal}
+type Catalog = Catalog of Season:string * Items:(CatalogItem [])
+type Order = Order of string[]
+type Receipt = Receipt of string
```

Comment:
If a receipt would be list of strings to represent receipt lines then you can use comparison on lists which will go item by item and will mean a reasonable result. Moreover, you will be able to demonstrate how exactly the doc string is parsed. I guess that it is the reason why the test is failing right now. Or you can use just string but then it is necessary that the "ordering step" will generate correct recipe without additional whitespaces.